Automattic/jetpack
# Launchpad: Add new task list for post-launch in Customer Home

Fixes https://github.com/Automattic/wp-calypso/issues/77279

## Proposed changes:

* Register a new task list called "Keep building". This is just a skeleton; we'll be adding tasks and an `is_enabled_callback` in the coming weeks.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sandbox `public-api.wordpress.com` and apply this patch to your sandbox
* Visit the developer console
* From the WP REST API GET `wpcom/v2/sites/:siteSlug:/launchpad?checklist_slug=keep-building`
* You should receive a response like this, a list of one task, "setup_general": 

<img width="962" alt="Screen Shot 2023-05-25 at 1 34 45 PM" src="https://github.com/Automattic/jetpack/assets/2124984/db734679-95f6-4ea8-9427-5c980ea59c4a">
